Bottom: 872decc47d16f64fd797f2dff066864b99974ac9
Top:    f1c6e3fc5ab998163f0474d1143aa5e31ee6d1bb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 16:51:26 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
